sagemaker-pipeline-multi-model/restate-project.ipynb

```diff
@@ -0,0 +1,787 @@
+{
```

Comment:
<p>May be more helpful to put the exact name of the kernel</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3445/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Committed requested change to yacatab:sagemaker-pipeline-multi-model. Thank you!